lib/streamlit/elements/doc_string.py

```diff
@@ -106,6 +114,10 @@ def help(self, obj: Any = streamlit) -> "DeltaGenerator":
         >>>
         >>> # Get help for Streamlit itself:
         >>> st
+
+        .. output::
+            https://doc-string2.streamlit.app/
+            height: 700px
```

Comment:
This last example app is using columns, while the code above is not.

Can you remove st.columns from the app?

Reply:
The only reason I included columns in the app is that both apps [1](https://doc-string.streamlit.app/) and [2](https://doc-string2.streamlit.app/) would look identical to users who don't scroll down a whole page in app 2 to get to the `st`  docstring. Could you 👍 to confirm I should still remove the columns?

Reply:
Oh, I see. How about we replace 

```
>>> # Get help for Pandas DataFrame:
>>> pandas.DataFrame
```

with 

```
>>> # Get help for Pandas read_csv:
>>> pandas.read_csv
```

Reply:
Good idea! Fixed on 87d332f and [14dd793](https://github.com/streamlit/docs/commit/14dd79329d530900c64b743ef46852c378f0993b) + updated the screenshot in the PR description. Will merge once the tests pass.